src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_cluster.py

```diff
@@ -38,11 +37,11 @@ def test_cluster_certs(self):
         self.cmd('keyvault certificate create --vault-name {kv_name} -n {cert_name2} -p @"{policy_path}"')
 
         while True:
-            cert = self.cmd('keyvault certificate show --vault-name {kv_name} -n {cert_name}').get_output_in_json()
-            if cert:
+            cert2 = self.cmd('keyvault certificate show --vault-name {kv_name} -n {cert_name2}').get_output_in_json()
```

Comment:
I am curious about why rename cert to cert2

Reply:
just for readability, this cert is used for adding as secondary certificate so I renamed it to cert2